deploy/vagrant/conf/alluxio.yml

```diff
@@ -4,13 +4,16 @@
 # {Github | Local | Release}
 Type: Local
 
+# If the version is less than 1.1.0, ie. 1.0.1. This should be false for 1.1.0-SNAPSHOT. {true|false}
+Version_LessThan_1.1.0: false
```

Comment:
here is a oneliner bash command that returns true of the version is at least 1.1: `./bin/alluxio version | sed -e 's/^.*version:\ \(...\).*$/\1/' | { IFS=. read major minor; if [[ $major -ge 1 && $minor -ge 1 ]]; then true; else false; fi }`